### PR TITLE
[Examples] Fix component scenes

### PR DESCRIPTION
**Pull request by untereiner**
xml header, solver iterations, etc...

______________________________________________________

This PR: 
- [x] builds with SUCCESS for all platforms on the CI.
- [x] does not generate new warnings.
- [x] does not generate new unit test failures.
- [x] does not generate new scene test failures.
- [x] does not break API compatibility.
- [x] is more than 1 week old (or has fast-merge label).

**Reviewers will merge only if all these checks are true.**

### CODE REVIEW

**Comment by damienmarchal** [comment]
Many thanks for all your PR @untereiner 

This looks simple and I trust you it is the thing to do. So let's wait our 7 days quarantine if someone says a 'no-go' and merge it otherwise.

File: examples/Components/topology/RegularGridTopology.scn [hunk 1]
```diff
@@ -23,10 +23,10 @@
             <Sphere name="Surf" listRadius="@[-2].listRadius" />
             <BarycentricMapping input="@.." output="@." />
         </Node>
-    </Node>
+    </Node> -->
```
Comment (guparan) [inline examples/Components/topology/RegularGridTopology.scn]: Why did you comment this block?

Comment (untereiner) [reply examples/Components/topology/RegularGridTopology.scn]: a mistake, I will revert these comments soon

Comment (damienmarchal) [reply examples/Components/topology/RegularGridTopology.scn]: @untereiner Can you do that before wedsneday so that we can merge your PR ?

Comment (untereiner) [reply examples/Components/topology/RegularGridTopology.scn]: sorry I couldn't... I am doing it now

**Comment by damienmarchal** [comment]
[ci-build]

**Comment by damienmarchal** [comment]
[ci-build]

**Comment by damienmarchal** [comment]
[ci-build]

**Pull request merged**